integration/loader/main.go

```diff
@@ -54,9 +53,6 @@ func main() {
 
 	chainID := flowsdk.ChainID([]byte(*chainIDStr))
 
-	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
you should likely leave the top level context -- we'll need it for multiple things. (for example it is still useful for the proper pusher shutdown.)

Reply:
i disagree.  this does nothing right now.  we can always add it back later if pusher does eventually support proper shutdown

Reply:
I am fine with this either way, which would you both prefer? As the code is now without the context is slightly cleaner and we do have the history to reference when we want to readd it.